### PR TITLE
Freon fixes, tweaks and balancing

### PR DESCRIPTION
**Pull request by Ghilker**
## About The Pull Request
Some fixes, bugs and balancing on all freon stuff, see changelog for what's new
Changed the minimum freon temperature cap to 50 K because there were negative heat (and pressure) spikes when reacting due to bad capping and too steep energy usage, that's now slower and the temperature go down in a nice pace. No more negative temperatures

## Why It's Good For The Game
fix and balancing is good

## Changelog
:cl:
add: Added cargo worth for freon
tweak: changed the value of hot ice (increased to mythril level)
tweak: increased the amount of plasma released from hot ice to 150 moles per sheet
tweak: increased the amoung of plasma (chem) from hot ice to 300 moles per sheet in grinding
tweak: changed hypernob canister sprite to differenciate it from the freon one
balance: made hot ice easier to make (range of temperatures to 120-160 K and halved the rate of cooling of freon/o2 reaction)
fix: fixed hot ice not showing up over 34 sheet in the stack
fix: fixed negative temperature issue caused by bad coding the cap, increasing the minimum temperature and changed the energy used by the reaction
/:cl:

### CODE REVIEW

File: code/modules/atmospherics/gasmixtures/reactions.dm [hunk 1]
```diff
@@ -281,10 +281,10 @@ datum/gas_reaction/freonfire/react(datum/gas_mixture/air, datum/holder)
 			ASSERT_GAS(/datum/gas/carbon_dioxide,air)
 			cached_gases[/datum/gas/carbon_dioxide][MOLES] += freon_burn_rate
 
-			if(temperature < 150 && temperature > 130 && prob(2))
+			if(temperature < 160 && temperature > 120 && prob(2))
 				new /obj/item/stack/sheet/hot_ice(location)
 
-			energy_released += FIRE_FREON_ENERGY_RELEASED * (freon_burn_rate)
+			energy_released += FIRE_FREON_ENERGY_RELEASED * (freon_burn_rate) * 0.45
```
Comment (AnturK) [inline code/modules/atmospherics/gasmixtures/reactions.dm]: magic number, why is this not incorporated in the first define ?

**Review by AnturK — CHANGES_REQUESTED** [review]

File: code/modules/atmospherics/gasmixtures/reactions.dm (hunk 1 above)
Comment (Ghilker) [reply code/modules/atmospherics/gasmixtures/reactions.dm]: good question! i don't know! i'll change it in the next commit

Comment (Ghilker) [reply code/modules/atmospherics/gasmixtures/reactions.dm]: changes done, made the cap a bit higher too just to be safe

**Comment by TheSilverNuke** [comment]
Great! Can't wait to try out these new improvements.

**Pull request merged**